src/blog/v8-release-85.md

```diff
@@ -0,0 +1,126 @@
+---
+title: 'V8 release v8.5'
+author: 'Sathya Gunasekaran'
+avatars:
```

Comment:
Add your avatar to `img` folder in 2 sizes and reference here.

Reply:
@zeynepCankara Only the images in `src/_img/avatars/*` are needed; please remove the ones you added in `src/_img/*.jpg`. Also, could you check the dimensions of the avatar images (it doesn't look like they're square)? They should match the other avatars.